src/styles/utils/normalize.scss

```diff
@@ -0,0 +1,11 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
```

Comment:
Use of flex properties on the body tag might not have the desired effect when content exceeds the viewport height. Consider using these properties on a specific container.